core-bundle/src/Resources/contao/widgets/KeyValueWizard.php

```diff
@@ -139,11 +139,11 @@ public function generate()
 			{
 				if ($button == 'drag')
 				{
-					$return .= ' <button type="button" class="drag-handle" title="' . StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['move']) . '" aria-hidden="true">' . Image::getHtml('drag.svg') . '</button>';
+					$return .= ' <button type="button" class="drag-handle" title="' . StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['move']) . '" aria-hidden="true">' . Image::getHtml('drag.svg', $GLOBALS['TL_LANG']['MSC']['move']) . '</button>';
```

Comment:
I don't like these changes. We have deliberately not added the button title to the button image as well, because screen readers will read it twice otherwise. Why is this necessary?

Reply:
Unfortunately, the current tooltip implementation removes the title attribute on init, and so we cannot simply copy the previous line and call `addInteractiveHelp` again. I thought it would be usefull to have the text also as `alt` and copy it before tooltip init from the button to the link (`copyInteractiveHelpFromButtonImage`). If this is a bad solution for screen readers, we should choose a different way!

Of course we could also add another `data` attribute (to be defined!) with the text instead of using the `alt` of the image and then copy the help from there or we need to change the tooltip init!

Reply:
According to the source code, the attribute is stored as `tip:native`: https://github.com/mootools/mootools-more/blob/687363b141c7d6abb89e6716462bbd99545f81e5/Source/Interface/Tips.js#L134 Maybe we can retrieve the value this way?

Reply:
Thanks for the hint! I could really retrieve the help before the clones and refactored the PR!